spec/index.bs

```diff
@@ -158,7 +158,17 @@ Type Definitions {#type-definitions}
 ============================
 
 <script type=idl>
-typedef unsigned long long GPUBufferSize;
+typedef [EnforceRange] unsigned long long GPUBufferSize;
+typedef [EnforceRange] unsigned long GPUBufferDynamicOffset;
+typedef [EnforceRange] unsigned long long GPUFenceValue;
+typedef [EnforceRange] unsigned long GPUStencilValue;
+typedef [EnforceRange] unsigned long GPUSampleMask;
+typedef [EnforceRange] long GPUDepthBias;
+
+typedef [EnforceRange] unsigned long GPUIntegerCoordinate;
+typedef [EnforceRange] unsigned long GPUIndex32;
+typedef [EnforceRange] unsigned long GPUSize32;
+typedef [EnforceRange] long GPUSignedOffset32;
```

Comment:
Note: I tried not to go too overboard with the number of different types, so consolidated some of them into these generic ones.